environments/india/public.yml

```diff
@@ -7,8 +7,8 @@ DATADOG_ENABLED: True
 elasticsearch_endpoint: '{{ groups.elasticsearch.0 }}'
 #elasticsearch_memory: '4096m'
 elasticsearch_cluster_name: 'indiahqes-1.x'
```

Comment:
Should we change the `elasticsearch_cluster_name` too?

Reply:
Not really sure, what all will it impact.

Reply:
Not sure it is a good idea at this point of time.